tests/Db/OldPasswordMapperTest.php

```diff
@@ -207,4 +209,31 @@ public function testGetPasswordsAboutToExpireAllMarked() {
 		$this->assertSame($uid, $latestPassword->getUid());
 		$this->assertLessThan($baseTime + 130, $latestPassword->getChangeTime());
 	}
+
+	public function testCleanUserHistory() {
```

Comment:
I don't think this test does what we want it to do.
The method should delete specific DB entries; what you're testing here is that the query builder is called in a predefined way. It seems 2 different things to me.

If I remove the `execute` statement, the test won't fail, but the method won't do what is supposed to do.
If I change the implementation to use a raw SQL statement (because random reasons), even if it works properly, the test will fail. Why the test should complain when the user entries are deleted?

Reply:
Modified the test. Now the test has 3 users. And removed one of them. Verified the removal and existence of other users.